README.md

```diff
@@ -1 +1,44 @@
-# React_redux_capstone_project
\ No newline at end of file
+## COVID 19 top 20 African cases
+
+> This is a website that will display top 20 African countries that are most affected by COVID. A rest API has been used to fetch the COVID data of all the listed countries.
+
+## Screenshot
+
+![screenshot](/src/images/screenShot.PNG)
+
+## Live Demo
+
+- [COVID African Data](https://dapper-biscuit-0dbbad.netlify.app/)
+
+## Video presentation
+
+[Video Presentation Link](https://www.loom.com/share/81bdfd376c0a4653a059635b769f30cb)
+
+## Built With
+
+- React
+- Redux
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone the repository by running git clone `https://github.com/Micky373/React_redux_capstone_project.git`
+- Navigate to the folder `cd React_redux_capstone_project`
+- Install packages by running `npm install`
+- Run the app by running `npm start`
+
+## Author
+
+👤 **Michael Tamirie**
+
+- GitHub: [Michael Tamirie](https://github.com/Micky373)
+- LinkedIn: [Michael Tamirie](https://www.linkedin.com/in/michael-tamirie-288a331ab)
+
+## Acknowledgments
+
+- Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
+
+## Show your support
+
+Give a ⭐ if you like this project!
```

Comment:
- [ ] Please let's include the License section in your README file. This will render our README file professional.